packages/generators/admin/component/index.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const { join } = require('path');
+const { flow, camelCase, upperFirst, lowerCase } = require('lodash');
+const fileExistsInPackages = require('../utils/fileExistsInPackages');
+const getPluginList = require('../utils/getPluginList');
+const packagesFolder = require('../utils/packagesFolder');
+
+const pascalCase = flow(camelCase, upperFirst);
+
+const prompts = [
+  {
+    type: 'list',
+    name: 'plugin',
+    message: 'Which plugin should be targeted?',
```

Comment:
Do you think we should translate these messages?

Reply:
If we plan to make those generators available to the community, I would say yes.

Reply:
I don't think it's a priority right now, our codebase is in english